### PR TITLE
Addonifies everything

### PR DESCRIPTION
**Pull request by pzuraq**
@cyrilsf

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,11 +0,0 @@
-Copyright © 2012 Addepar, Inc. All Rights Reserved.
```
Comment (cyril-sf) [inline LICENSE]: We should keep that file

Comment (cyril-sf) [inline LICENSE]: Or ask legal

Comment (pzuraq) [inline LICENSE]: I'll put it back, it's easier

File: README.md [hunk 2]
```diff
@@ -1,108 +1,25 @@
-# Ember Widgets by Addepar [![Build Status](https://secure.travis-ci.org/Addepar/ember-widgets.svg?branch=master)](http://travis-ci.org/Addepar/ember-widgets)
```
Comment (cyril-sf) [inline README.md]: I would keep and update the original file

Comment (pzuraq) [inline README.md]: The original file is basically all references to Grunt stuff, I updated it a little but forgot to commit the changes, will push in a sec

File: bower_components/bootstrap/less/mixins/list-group.less [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// List Groups
```
Comment (cyril-sf) [inline bower_components/bootstrap/less/mixins/list-group.less]: we probably don't want to commit `bower_components`

**Comment by pzuraq** [comment]
@cyril-sf ready for merge

**Comment by cyril-sf** [comment]
LGTM

**Pull request merged**